react/week2/todo-part2/src/App.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from "react";
+import ToDos from "./ToDos";
+
+const Header = () => {
+  return <h1> Todo List</h1>;
+};
+
+const Timer = () => {
+  const [count, setCount] = useState(0);
+  useEffect(() => {
+    setTimeout(() => {
+      setCount((timer) => timer + 1);
+    }, 1000);
+  });
+  return <h3>You have used {count} seconds on this website</h3>;
+};
+// component to check the todo array when it is empty render a message
+const CheckTodoArray = (props) => {
+  if (props.todoArray.length === 0) {
+    return <div> No Items </div>;
+  } else {
+    return null;
+  }
+};
```

Comment:
this could/should be done in the HTML using the conditional ternary operator

`{todoArray.length === 0 && <div>No Items</div>}`

Essentially, what this means is only if true `&&` show the `<div>No Items</div>`